modules/income_and_assets/lib/income_and_assets/helpers.rb

```diff
@@ -32,7 +32,7 @@ def format_date_to_mm_dd_yyyy(date_string)
     # @return [Hash]
     #
     def split_currency_amount_sm(amount, field_lengths = {})
-      return {} if !amount || amount.negative? || amount >= 100_000
+      return {} if !amount || amount.negative? || amount >= 1_000_000
```

Comment:
I assume that the max value here is supposed to be 999,999.99 instead of 99,999.99